static/src/components/task_browse/assign_worker.vue

```diff
@@ -0,0 +1,129 @@
+<template>
+  <div class="form-horizontal">
+    <div class="form-group">
+      <div class="col-sm-12">
+        <GigSpinner v-if="waiting" />
+        <div>
+          <label class="typo__label">Assign users</label>
+          <multiselect
+            v-model="addUserValues"
+            placeholder="Search"
+            label="fullname"
+            track-by="email"
+            :options="allUserList"
+            :multiple="true"
+          />
+        </div>
+        <br>
+        <div>
+          <label class="typo__label">Remove users</label>
+          <multiselect
+            v-model="removeUserValues"
+            placeholder="Search"
+            label="fullname"
+            track-by="email"
+            :options="assignedUserList"
+            :multiple="true"
+          />
+        </div>
+      </div>
+    </div>
+    <p>
+      Note:
+      Type in users to assign them to the selected tasks
```

Comment:
This UI provides option to add and/or remove users. Note can be updated describe both user assignment and removal.